Scripts/NewBoopScript.js

```diff
@@ -0,0 +1,52 @@
+/**
+	{
+		"api":1,
+		"name":"New Boop Script",
+		"description":"Returns a basic Boop script.",
+		"author":"tlewis",
+		"icon":"quote",
+		"tags":"boop,state,script,debug,new,create"
+	}
+**/
+
+function main(state) {
+	try {
+		state.text = script
+	}
+	catch(error) {
+		state.postError("Something strange happened here...")
+	}s
```

Comment:
```suggestion
    }
```

Extra character... Happens to me all the time :D